_episodes/02-installation.md

```diff
@@ -1,119 +1,192 @@
 
-To create an environment, go to the directory containing the ESMValTool source
-code (called ESMValTool if you did not choose a different name) and run
+2.  Next, run the installer
 
-~~~
-    conda env create --name esmvaltool --file environment.yml
-~~~
-{: .source}
+    ```bash
+    bash Miniconda3-latest-MacOSX-x86_64.sh
+    ```
 
-The environment is called ``esmvaltool`` by default, but it is possible to use
-the option ``--name ENVIRONMENT_NAME`` to define a custom name. You can activate
-the environment using the command:
+3.  Follow the instructions in the installer. The defaults should normally suffice.
 
-~~~
-    conda activate esmvaltool
-~~~
-{: .source}
+4.  You will need to restart your terminal for the changes to have effect.
 
-If you run into trouble, please try recreating the environment. More information can be found in the (conda documentation)[https://docs.conda.io/en/latest/].
+5.  Verify you have a working conda installation by listing all installed packages
 
+    ```bash
+    conda list
+    ````
 
-## Software installation
+### Windows
 
-Once all prerequisites are fulfilled, ESMValTool can be installed by running
-the following commands in the directory containing the ESMValTool source code:
+ESMValTool does not directly support Windows, But succesful usage has been reported through the [Windows Subsystem for Linux(WSL)](https://docs.microsoft.com/en-us/windows/wsl/), available in Windows 10.
 
-~~~
-    pip install .
-~~~
-{: .source}
+To install the WSL please follow the instructions [on the Windows Documentation page](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
 
-## Test the installation
+After installing the WSL, installation can be done using the Linux installation instructions.
 
-The next step is to check that the installation works properly.
-To do this, run the tool with:
+## Install Julia
 
-~~~
-    esmvaltool --help
-~~~
-{: .source}
+Complete instructions for installing Julia can be found on the
+[Julia installation page](https://julialang.org/downloads/platform/#linux_and_freebsd).
+In this tutorial, we will use the following steps.
+First, open a bash terminal and create a directory to install Julia in
 
-If everything was installed properly, ESMValTool should have printed a
-help message to the console.
+```bash
+mkdir ~/julia
+```
 
-For a more complete installation verification, run the automated tests and
-confirm that no errors are reported:
+Next, download the file
+[`julia-1.0.5-linux-x86_64.tar.gz`](https://julialang-s3.julialang.org/bin/linux/x64/1.0/julia-1.0.5-linux-x86_64.tar.gz)
+by clicking the link or going to the [Julia downloads page](https://julialang.org/downloads/).
+To extract the file, you can use the following command:
 
-~~~
-    python setup.py test --installation
-~~~
-{: .source}
+```bash
+tar -xvzf ~/Downloads/julia-1.0.5-linux-x86\_64.tar.gz -C ~/julia
+```
 
-{% include links.md %}
+This will extract the files to a folder named `~/julia/julia-1.0.5`.
```

Comment:
Maybe it would be nice to verify that this file is there?

Reply:
Added download location, is this clearer?